### ISSUE TITLE
"Display Field in Selection List" doesn't work for displaying relationship taxonomy field

### ISSUE THREAD

**Issue by CPapadim** [pods-framework/pods#4581]
In Pods 2.7

As the title suggests, if you specify a relationship field to display, you get back a SQL error.  Based on the error it seems like it either looks for the field's data in the wrong table (e.g. in the table for the pod that has the relationship field), and is unable to find the data, or perhaps taxonomies have different column names than other posts for storing data and so is looking for a non-existing column.

I haven't tested this with a relationship field to a regular post type that isn't a taxonomy so not sure if that works.

**Comment by sc0ttkclark** [comment]
That's correct, currently only a very limited number of fields can be used with this, depending on the related object type. For taxonomy, I believe this is name, term_id, parent, or description. Sorry about that, we do plan on expanding it when Pods 2.8 hits with it's brand new PodsObject class structure that allows for anything/everything to act the same exact way. Meaning -- every relationship, even if it's not related to another pod, can be interacted with like another pod.

**Comment by jimtrue** [comment]
As a workaround, if you extend the Taxonomy on the other side of the relationship with Pods, this might solve your problem. You don't have to add any fields to it, just extend it with Pods so that Pods is aware of it.

**Comment by CPapadim** [comment]
It's actually a pods taxonomy.  It has a relationship field in one of my regular post-type (non-taxonomy) pods, and then I want to use it for that pod, is when I get the error.  So when using {@field_name} for it.

Can I do something like {@filed_name.name}?
